src/DynamoRevit/DynamoRevit.cs

```diff
@@ -27,6 +27,7 @@
 using DynamoInstallDetective;
 using Greg.AuthProviders;
 using Microsoft.Win32;
+using Newtonsoft.Json;
```

Comment:
@BogdanZavu looks like the 2017 branch is missing the using statement for json.net